google/youtube/tool.gpt

```diff
@@ -1,3 +1,9 @@
+---
+Name: YouTube
+Description: Tools for interacting with YouTube
+Share Tools: Transcribe Youtube Video, Youtube Transcriber, Advanced Youtube Transcriber
```

Comment:
Not sure folks will use `Youtube Transcriber` on its own, so maybe it doesn't make sense to include it in the bundle.

Reply:
I've played around with this to make all the tools useful in some capacity.